### ISSUE TITLE
5 anti-UC hate changes

### ISSUE THREAD

**Issue by NoProblemWithThisAcc** [captaincobbs/ESG-Mod#1415]
tl;dr
0) UC is vulnerable to meta-gaming:

1) UC citadels need to be hidden
2) Military tech->Science tier ghost cloaking
3) UC cloaking SD construction->availability
4) 1 or 2 planet systems should be hide-able
5) UC sleepers need to generate, not steal

x) None of this really changes balance ... yet. We can discuss crescents, backdoors, etc, later. Full text if you find yourself disagreeing with any of those five things, or just want the context:

UC needs an overhaul. If the player base is going to consider it not cheating to call out in all-chat where the hidden player so they can 1v4 instead of 1v1v1v1v1, then as a non-exploit it needs fixed. I played almost completely perfect game Saturday, under a time limit and was less than half of the Vaulter's score. He had 2k science, I had almost a tenth of that (despite 3 science behemoths). Perhaps I could have run the game for another 3 hours and caught up, but I don't think so, nor do I think Vaulters are too powerful. In fact, I don't want to because such a perfect game setup will probably never happen again (considering the current UC mechanics): I had an AI next to me that I could have for sleepers, but who only colonized after my sanctuaries were in place, as well as a Hissho player that never colonized so I could fill up the entire quarter of the map with sanctuaries. He was also clearing pirates for me.

I went full greed mode: no military, all science, food and industry, as wide as possible while still ecstatic, maxing population to food planets, rushing behemoths, etc. I was about 700 score when we called it. I was behind the AI and only noticeably ahead of the Horatio because he disconnected. It's true that I did have a poor start, and rough planets nearby (gas, ice, lava, swamp) and I was a level or two lower on my hero than I could have been, but I managed to snag a minor before anyone noticed. I decided to re-run a test vs 4 AI's on a medium twin galaxy using my usual settings but with no time limit and after _restarting several times_ for a good start with lots of food-rich planets nearby. I managed 900 score by turn 32 without me worrying about being detected. It's still 800 +/- 100 which was about where the Hissho player (who was asking for advice during the game) was.

Both runs were without being found out on turn 15-20 which is about when a Riftborn player could scout a level 2 cloaked home system and then ship 5 hunters off to it a few turns later. Yes I've had that happen to me in multiplayer, due to meta-game knowledge about where the UC homeworld likely was. This Saturday, I did everything I could to make sure no one would ever know where I was (such as not ever taking detectable hacks), all without slowing down my gameplay mostly due to intense luck. And in spite of this effort, all 4 players were actively hunting me in a coordinated effort 1v4-style. Due to chaotic empire placement, they were all certain that I was on the other side of the galaxy, which is the only reason I was almost able to keep up with the unsure Hissho player.

Also, I didn't citadel as this would be an immediate red flag of where I was located. This is a visual-only thing and should be removed, even if it means removing the ability to see where citadels are for all other players (increasing the memory tax of other players). Alternately, another constructible could be created direct that costs a behemoth with the same stats as a base citadel.

UC homeworld cloak should be off the ABILITY to SD up, not actually having the resources to invest in an immediate SD. This means that even if you don't have 100 t2's available (or 300 t1's), you'll still be safe early midgame, and the same thing for the second SD upgrade late midgame.

Furthermore cloaking upgrades should be by science tiers for ghost. There's simply no way to keep up with full colonization races while investing in t3 or t4 military while also in t3 and t4 science. This won't change anything in the UC playstyle, since it already hinges on just hoping you successfully avoid both players and pirates while going full greed, nor will it imbalance anything since you'll still be half the score of the better players. Note that this was proposed years ago, but just didn't get done. If it's was 'sure, you go do it,' then I'll just add it to my list.

Single planet systems should start out at cloaking level 3, and 2 planet systems should start out at cloaking level 2 (or get level 3 upon re-hacking) so you can take them, if you can afford the extra-punishing overcolonization, just like a normal race but without blowing your cover. Sure, they are still suboptimal because a future colonizing race won't have as much pop growth and therefore chances for sleepers (umbral shadows upon invasion), but it's better than having a giant noob trap to players who don't know to decolonize them (and carefully avoid ever letting population spill over there, as those pops will instantly die as has happened to me in a MP match before).

I think the reason all players are instantly hostile to even the mere thought of an umbral choir player isn't the threat of desyncs (which doesn't seem to happen if the UC player is the host or has good internet). Its because players don't want to lose luxuries, strategics, and dust without killing their own pop or losing happiness to find sleepers. Considering UC is a pacifist quest line race, the solution is obvious: don't make players lose those resources to UC sleepers. UC sleepers can magically make those resources copy over to the UC player, without being lost.

We should make the UC into a commensalist race rather than a parasite. This makes UC like most every other race. If there's a Riftborn near me doing his Riftborn things, this doesn't hurt my empire. But having a UC around means there are now landmines I might step on as a non-UC player, and thus I'd have to clear them out to avoid being hurt. Sure the UC player might declare war and attack later, but so could any other race (and a Vodyani could do it without declaring war). De-escalating the UC hate can only be good for the player base, not bad. Of course, this doesn't stop a Riftborn who wants to ruin their own game by rushing detection 3 and hunters, but almost every FFA game suffers from that possibility.

**Comment by Arpharnator** [comment]
yea no
